tests/x11/updates_gnome.pm

```diff
@@ -0,0 +1,88 @@
+# SUSE's openQA tests
+#
```

Comment:
In the complete file: You are using tabs instead of 4 space indention. Fix your editor!